src/wp-includes/customize/class-wp-customize-nav-menu-item-control.php

```diff
@@ -77,10 +77,22 @@ public function content_template() {
 				</span>
 				<span class="item-controls">
 					<button type="button" class="button-link item-edit" aria-expanded="false"><span class="screen-reader-text">
-					<?php
-						/* translators: 1: Title of a menu item, 2: Type of a menu item. */
-						printf( __( 'Edit menu item: %1$s (%2$s)' ), '{{ data.title || data.original_title || wp.customize.Menus.data.l10n.untitled }}', '{{ data.item_type_label }}' );
-					?>
+					<# if ( 0 === data.depth ) { #>
+						<?php
+						/* translators: 1: Title of a menu item, 2: Type of a menu item */
+						printf( __( 'Edit %1$s (%2$s, %3$d of %4$d)' ), '{{ data.title || data.original_title || wp.customize.Menus.data.l10n.untitled }}', '{{ data.item_type_label }}', '', '' );
+						?>
+					<# } else if ( 1 === data.depth ) { #>
+						<?php
+							/* translators: 1: Title of a menu item, 2: Type of a menu item, 3: Depth of a menu item */
+							printf( __( 'Edit %1$s (%2$s, sub-item %3$d of %4$d of %5$s)' ), '{{ data.title || data.original_title || wp.customize.Menus.data.l10n.untitled }}', '{{ data.item_type_label }}', '', '', '' );
```

Comment:
Change to `under %5$s`. Better English grammar, makes the string easier to parse while placeholders are present.

Also update the translator comment.